src/EditorFeatures/CSharpTest/GenerateVariable/GenerateVariableTests.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.CodeAnalysis.CodeActions;
 using Microsoft.CodeAnalysis.CodeFixes;
 using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp;
```

Comment:
Not related to this PR, I've been thinking about code fixers and LangVersion, as the topic came up in a [recent issue](https://github.com/dotnet/roslyn/issues/26485).

I think the current behavior is that the fixer will just generate code, then once applied you will get offered to update your project... I wonder if we should have some kind of conflict on the fixer, to warn you that it will require a new LangVersion...

Reply:
Interesting idea.  Let's track that through an issue.  I think it should go through the design review.  @sharwell for eyes.

Reply:
Also not related to this PR, would you mind adding a test for `x is [|Type|] { }` where `Type` doesn't exist.
Please file issue if it doesn't trigger the "generate type" fixer.

Along those lines, should we have a "generate Deconstruct method" fixer? Type `var (x, y) = e; // no Deconstruct found` or `e is (1, 2)`. We could generate it for you. That'd be cool :-)
Filed https://github.com/dotnet/roslyn/issues/26597 for that

Reply:
Filed https://github.com/dotnet/roslyn/issues/26598 for general issue of code fixers and LangVersion.